runtime/v2/runc/options/oci.proto

```diff
@@ -19,19 +19,12 @@ message Options {
 	string binary_name = 6;
 	// runc root directory
 	string root = 7;
-	// criu binary path.
-	//
-	// Deprecated: runc option --criu is now ignored (with a warning), and the
-	// option will be removed entirely in a future release. Users who need a non-
-	// standard criu binary should rely on the standard way of looking up binaries
-	// in $PATH.
-	string criu_path = 8 [deprecated = true];
 	// enable systemd cgroups
-	bool systemd_cgroup = 9;
+	bool systemd_cgroup = 8;
```

Comment:
No, this breaks compatibility

Reply:
Sorry, I noticed the `reserved` statement,do we need to use it?:thinking:
https://protobuf.dev/programming-guides/proto3/#reserved

Reply:
Yes